src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -286,7 +286,7 @@ def _flexible_server_params(command_group):
                        help='The name of the new server that is created by the restore command.')
             c.argument('source_server', options_list=['--source-server'],
                        help='The name or resource ID of the source server to restore from.')
-            c.argument('restore_point_in_time', options_list=['--time'],
+            c.argument('restore_point_in_time', options_list=['--point-in-time'],
```

Comment:
I think the ask was to rename this to --restore-time for restore commands.  I have this in my PR, can you remove this here?